nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -126,6 +127,18 @@ public ValidationResult validate(final String subject, final String value, final
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
 
+    static final AllowableValue YES_PP = new AllowableValue("yes", "Yes");
+    static final AllowableValue NO_PP  = new AllowableValue("no", "No");
+    static final PropertyDescriptor USE_PRETTY_PRINTING = new PropertyDescriptor.Builder()
```

Comment:
In order to avoid making existing processors invalid or change the way that they behave, we should add a default value of "true" to this property. Then the user is still able to disable the pretty-printing, but those who are satisfied with the current behavior are not affected.